src/Music.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Music.h"
+
+#include <iostream>
+#include <algorithm>
+#include <cmath>
+
+namespace ccm
+{
+	Music::Music(std::filesystem::path filepath, int loops)
```

Comment:
Are `std::filesystem::path` iterator-like types that are cheap to copy, or should we consider passing these by const-reference?

Reply:
Yeah these should be const-ref. At first I had made them members of music and just std::moved the parameter, but figured there was no need. Will change it to const ref.